DOC update the deployment to include checklist (#427)
paris-saclay-cds/ramp-board: doc/create_ramp_event.rst

      more precisely the :ref:`conda workers <conda_env_worker>` to have more
      information.

Before you continue make sure that::
    1.  you have the private and the public data in the following directory

>         structure::

Reviewer: I think you need an empty line after the `::`

Also I think it would be better to have the bullet points outside of the 'code' block

>         structure::

Reviewer: I think you need an empty line after the `::`

Also I think it would be better to have the bullet points outside of the 'code' block

Reviewer: I think for bullet/numbered lists you need an empty line above the start of the list.

        ├──ramp_deployment
            ├── ramp-kits
                ├── iris
                    ├── data  # your public data
                        ├── test
                        ├── train
            ├── ramp-data
                ├── iris
                    ├── data  # your private data
                        ├── test
                        ├── train

> 

Reviewer: Maybe use the ` └──` notation for when the last file/dir

Reviewer: Also the file dir structure could be more clear with more lines, for example: https://paris-saclay-cds.github.io/ramp-docs/ramp-workflow/dev/workflow.html#overall-directory-structure

Reviewer: Actually this exact directory structure is not strictly necessary. This is why we changed to copying the data files from `ramp-data` to `ramp-kits` - to be robust to differences in directory structure. We suggest this structure in the `Deploy a specific RAMP event` section above with the commands given, but this exact structure is not required. It might be worthwhile to mention this.
Also for files add a `/` at the end, to indicate that they are files.
How the data files look can also vary. For the human locomotion challenge, the data were in `test` and `train` directories and the `get_train_data` and `get_test_data` were written to load files from a folder. For other challenges, it is common that the test and train data are just files within `data`. It would be good to make it clear that how the data files look can vary.

>     2.  conda environment used by your event exists (if you are using one)::

Reviewer: ```suggestion
    2.  The conda environment used by your event exists (if you are using one)::
```

Might be worth mentioning that the name of the conda env should match with that in the config.yml file (line 42)

      more precisely the :ref:`conda workers <conda_env_worker>` to have more
      information.

Before you continue make sure that:

    1.  you have the private and the public data in the following directory
        structure::

        └──ramp_deployment
            ├── ramp-kits
                └── iris
                    └── data  # your public data
                        ├── test
                        └── train
            └── ramp-data
                └── iris
                    └── data  # your private data
                        ├── test
                        └── train

    2.  conda environment used by your event exists (if you are using one). To
        check your conda environments::

>         > conda env list

Reviewer: Maybe use `$` to indicate shell?

      more precisely the :ref:`conda workers <conda_env_worker>` to have more
      information.

Before you continue make sure that:

>     1.  your private and public data are available.

Reviewer: ```suggestion
    1.  Your private and public data are stored in the correct folders.
```

        For example, your data might be in:

>         - public data: `ramp_deployment/ramp-kits/<problem_name>/data`, and
>         - private data: `ramp_deployment/ramp-data/<problem_name>/data`.

Reviewer: ```suggestion
        - public data: `ramp_deployment/ramp-kits/<problem_name>/data/`, and
        - private data: `ramp_deployment/ramp-data/<problem_name>/data/`.
```

>         Note that:

Reviewer: ```suggestion
        Note:
```

        - test and train can be directories or files, depending on the

>           challenge

Reviewer: You might not need this part anymore, now that we aren't using the directory structure example.

        - for more information on directory tree of the starting kits check

>           `overall-directory-structure

Reviewer: ```suggestion
          `Overall directory structure
```

          <https://paris-saclay-cds.github.io/ramp-docs/ramp-workflow/dev/workflow.html#overall-directory-structure>`_

>     2.  the conda environment (set in the ``config.yml`` file above, here

Reviewer: ```suggestion
    2.  The conda environment (set in the ``config.yml`` file above, here
```

>         called ramp-iris) used by your event exists. Note, that this only

Reviewer: ```suggestion
        called `ramp-iris`) used by your event exists. Note, that this only
```

>         applies to the events which use conda environment. To check which conda

Reviewer: ```suggestion
        applies to the events which use the conda environment. To check which conda
```

      more precisely the :ref:`conda workers <conda_env_worker>` to have more
      information.

Before you continue make sure that:

    1.  your private and public data are available. For example, you might have
        the following directory structure::

>         └──ramp_deployment/
>             ├── ramp-kits/
>                 └── iris/
>                     └── data/  # your public data
>                         ├── test
>                         └── train

Reviewer: ```suggestion
        └──ramp_deployment/
           ├── ramp-kits/
            |   └── iris/
            |       └── data/  # your public data
            |           ├── test
            |           └── train
```

Before you continue make sure that:

>     1.  your private and public data are available. For example, you might have

Reviewer: maybe:
'Your private and public data are in `ramp-kits/<my_challenge/data` and `ramp-data/<my_challenge/data` respectively.
